### PR TITLE
Update FabricCDN url to use latest filetype icon

### PR DESCRIPTION
**Pull request by bigbadcapers**
* [x] Code is up-to-date with the `master` branch
* [x] Changes were tested in storybook
* [x] You've run `yarn change` (and `yarn lage`) locally

### CODE REVIEW

**Review by spmonahan — APPROVED** [review]

File: change/@fluentui-style-utilities-6e2d4d2f-fb1b-4dc3-96ef-94dd267865f8.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-style-utilities-6e2d4d2f-fb1b-4dc3-96ef-94dd267865f8.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27779&commitId=9ad4c5e57bcc1c477e06088e791a952592b6207d&env=prod)

✅ There was 4 screenshots added, 0 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3aecae7fc8af9dc0c4de35f58a61b76ab9e9a659'>3aecae7fc8af9dc0c4de35f58a61b76ab9e9a659</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/BaseButton-mount_base.html">953</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/BaseButton-mount.html">1073</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Breadcrumb-mount_base.html">2683</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Breadcrumb-mount.html">2669</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Checkbox-mount_base.html">2616</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Checkbox-mount.html">2617</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/CheckboxBase-mount_base.html">2236</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/CheckboxBase-mount.html">2286</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ChoiceGroup-mount_base.html">4730</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ChoiceGroup-mount.html">4433</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ComboBox-mount_base.html">1109</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ComboBox-mount.html">1067</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/CommandBar-mount_base.html">9721</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/CommandBar-mount.html">9651</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ContextualMenu-mount_base.html">19441</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ContextualMenu-mount.html">19495</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/DefaultButton-mount_base.html">1158</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/DefaultButton-mount.html">1276</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/DetailsRow-mount_base.html">3398</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/DetailsRow-mount.html">3661</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/DetailsRowFast-mount_base.html">3534</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/DetailsRowFast-mount.html">3441</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/DetailsRowNoStyles-mount_base.html">3192</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/DetailsRowNoStyles-mount.html">3408</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Dialog-mount_base.html">4342</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Dialog-mount.html">4119</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/DocumentCardTitle-mount_base.html">416</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/DocumentCardTitle-mount.html">431</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Dropdown-mount_base.html">3194</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Dropdown-mount.html">3043</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/FocusTrapZone-mount_base.html">1921</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/FocusTrapZone-mount.html">2005</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/FocusZone-mount_base.html">1826</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/FocusZone-mount.html">1848</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedList-mount_base.html">68939</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedList-mount.html">68458</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedList-virtual-rerender_base.html">26400</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedList-virtual-rerender.html">28918</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">90306</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedList-virtual-rerender-with-unmount.html">91221</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedListV2-mount_base.html">432</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedListV2-mount.html">405</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedListV2-virtual-rerender_base.html">378</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedListV2-virtual-rerender.html">393</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">402</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">423</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/IconButton-mount_base.html">1652</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/IconButton-mount.html">1872</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Label-mount_base.html">593</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Label-mount.html">573</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Layer-mount_base.html">4413</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Layer-mount.html">4412</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Link-mount_base.html">661</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Link-mount.html">648</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/MenuButton-mount_base.html">1536</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/MenuButton-mount.html">1507</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/MessageBar-mount_base.html">33848</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/MessageBar-mount.html">33117</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Nav-mount_base.html">3142</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Nav-mount.html">3094</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/OverflowSet-mount_base.html">1206</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/OverflowSet-mount.html">1210</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Panel-mount_base.html">2851</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Panel-mount.html">2736</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Persona-mount_base.html">1341</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Persona-mount.html">1275</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Pivot-mount_base.html">1505</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Pivot-mount.html">1558</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/PrimaryButton-mount_base.html">1332</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/PrimaryButton-mount.html">1336</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Rating-mount_base.html">6978</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Rating-mount.html">7002</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/SearchBox-mount_base.html">1348</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/SearchBox-mount.html">1368</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Shimmer-mount_base.html">2768</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Shimmer-mount.html">2864</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Slider-mount_base.html">2258</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Slider-mount.html">2148</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/SpinButton-mount_base.html">4686</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/SpinButton-mount.html">4744</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Spinner-mount_base.html">625</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Spinner-mount.html">608</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/SplitButton-mount_base.html">2930</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/SplitButton-mount.html">2936</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Stack-mount_base.html">673</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Stack-mount.html">695</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/StackWithIntrinsicChildren-mount_base.html">1343</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/StackWithIntrinsicChildren-mount.html">1473</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/StackWithTextChildren-mount_base.html">4052</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/StackWithTextChildren-mount.html">3983</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/SwatchColorPicker-mount_base.html">9721</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/SwatchColorPicker-mount.html">10022</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/TagPicker-mount_base.html">2460</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/TagPicker-mount.html">2471</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Text-mount_base.html">585</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Text-mount.html">625</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/TextField-mount_base.html">1422</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/TextField-mount.html">1586</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ThemeProvider-mount_base.html">1421</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ThemeProvider-mount.html">1378</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ThemeProvider-virtual-rerender_base.html">1006</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ThemeProvider-virtual-rerender.html">954</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2045</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Toggle-mount_base.html">1066</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/Toggle-mount.html">1014</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/buttonNative-mount_base.html">359</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27779/perf-test/buttonNative-mount.html">381</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-style-utilities-6e2d4d2f-fb1b-4dc3-96ef-94dd267865f8.json (hunk 1 above)
Comment (bigbadcapers) [reply change/@fluentui-style-utilities-6e2d4d2f-fb1b-4dc3-96ef-94dd267865f8.json:1]: 4 new screenshots related to pivot control appeared, likely due to a prior PR but not connected to changes in this one. resolving this as ok.

**Comment by bigbadcapers** [comment]
Hey @micahgodbolt ! Happy fhl and qq for you. Do you think it's safe to authorize me and/or Peter to do merges when they affect this package or the Cdn.ts file? We regularly field requests for new filetype icons, and the PRs all pretty much follow this profile. Thanks (and lmk if there's a better channel for the question).

**Comment by bigbadcapers** [comment]
@spmonahan @jasoncuster @dzearing could you help with _merge_? i added an additional commit to this pending PR to resolve issue https://github.com/microsoft/fluentui/issues/27839 

cc @tiaxu1

**Pull request merged**